src/segments/zvm.go

```diff
@@ -0,0 +1,74 @@
+package segments
+
+import (
+	"strings"
+
+	"github.com/jandedobbeleer/oh-my-posh/src/properties"
+	"github.com/jandedobbeleer/oh-my-posh/src/runtime"
+)
+
+type Zvm struct {
+	props properties.Properties
+	env   runtime.Environment
+
+	Version string
+	ZigIcon string
+}
+
+func (z *Zvm) SetText(text string) {
+	z.Version = text
+}
+
+func (z *Zvm) Text() string {
+	return z.Version
+}
+
+const (
+	ZvmIcon properties.Property = "zigicon"
+)
+
+func (z *Zvm) Enabled() bool {
+	if !z.env.HasCommand("zvm") {
+		return false
+	}
+	version := z.getZvmVersion()
+	return version != ""
+}
+
+func (z *Zvm) Template() string {
+	return " {{ if .ZigIcon }}{{ .ZigIcon }} {{ end }}{{ .Version }} "
+}
+
+func (z *Zvm) Init(props properties.Properties, env runtime.Environment) {
+	z.props = props
+	z.env = env
+
+	// Initialize the icon from properties
+	z.ZigIcon = z.props.GetString(ZvmIcon, "ZVM")
+	z.Version = z.getZvmVersion()
+}
+
+func (z *Zvm) getZvmVersion() string {
+	output, err := z.env.RunCommand("zvm", "list")
```

Comment:
you can use `--color=false` to not have ANSI in the output.

Reply:
Changed command to disable color output before if it is enabled, updated parsing logic to look for '[x]' that marks version instead of color codes, and it now makes an attempt to return color settings to whatever the user had after grabbing the version.